src/last.html

```diff
@@ -0,0 +1,54 @@
+<!--
```

Comment:
Can we find the next sample via the filesystem instead? I'm currently working on adding support for sections which requires examples being put into different subfolders. The last marker via file might no longer work.